certified-connectors/Eigen Events Connector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,298 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Eigen Events",
+    "description": "Provides events from Ingenuity for use as flow triggers. You can use this connector to start flows whenever a selected event completes in the Eigen Ingenuity System",
+    "x-ms-api-annotation": {
+      "status": "Preview"
+    },
+    "contact": {
+      "email": "support@eigen-ltd.com",
+      "name": "Eigen Ltd",
+      "url": "https://www.eigen.co/"
+    }
+  },
+  "host": "c91a5acf572e.ngrok.io",
+  "basePath": "/",
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/events/api/webhook/{eventIds}": {
+      "x-ms-notification-content": {
+        "description": "Details of the incoming event.",
+        "schema": {
+          "type": "object",
+          "x-ms-dynamic-schema": {
+            "operationId": "GetDynamicEventSchema",
+            "parameters": {
+              "eventIds": {
+                "parameter": "eventIds"
+              }
+            },
+            "value-path": "schema"
+          }
+        }
+      },
+      "post": {
+        "description": "This operation triggers when any of the events subscribed to occurs",
+        "summary": "When an event occurs",
+        "operationId": "CreateWebhook",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "eventIds",
+            "in": "path",
+            "required": true,
+            "x-ms-summary": "EventID",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "GetSupportedEvents",
+              "value-path": "value",
+              "value-collection": "events",
+              "value-title": "display"
+            },
+            "description": "Select Event Id",
+            "type": "number"
+          },
+          {
+            "name": "Body of Trigger Request",
+            "in": "body",
+            "description": "The body of Trigger-Creation request",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required": [
+                "callbackUrl"
+              ],
+              "properties": {
+                "callbackUrl": {
+                  "title": "The callback URL",
+                  "x-ms-summary": "Callback URL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Response to successfully created webhook.",
+            "schema": {
+              "$ref": "#/definitions/WebhookCreationResponse"
+            }
+          },
+          "default": {
```

Comment:
@tonyscofieldkim Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Refer - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-responses)